statistics/histogram.go

```diff
@@ -1211,7 +1211,7 @@ func (c *Column) GetColumnRowCount(sctx sessionctx.Context, ranges []*ranger.Ran
 		// `betweenRowCount` returns count for [l, h) range, we adjust cnt for boundaries here.
 		// Note that, `cnt` does not include null values, we need specially handle cases
 		// where null is the lower bound.
-		if rg.LowExclude && !lowVal.IsNull() {
+		if rg.LowExclude && !lowVal.IsNull() && lowVal.Kind() != types.KindMaxValue && lowVal.Kind() != types.KindMinNotNull {
```

Comment:
Please add some comment to explain in what case the condition `lowVal.Kind() != types.KindMaxValue && lowVal.Kind() != types.KindMinNotNull` can be true.

Reply:
Updated.